test/controllers/presroc/bill_runs_invoices.controller.test.js

```diff
@@ -38,22 +37,19 @@ const { BillRunModel, InvoiceModel } = require('../../../app/models')
 const JsonWebToken = require('jsonwebtoken')
 
 describe('Presroc Invoices controller', () => {
-  const clientID = '1234546789'
   let server
   let authToken
   let regime
-  let authorisedSystem
-  let billRun
   let invoice
 
   beforeEach(async () => {
     await DatabaseHelper.clean()
     server = await init()
 
     regime = await RegimeHelper.addRegime('wrls', 'WRLS')
-    authorisedSystem = await AuthorisedSystemHelper.addSystem(clientID, 'system1', [regime])
-    billRun = await BillRunHelper.addBillRun(authorisedSystem.id, regime.id)
-    invoice = await InvoiceHelper.addInvoice(billRun.id, 'CUSTOMER', '2020')
+    await AuthorisedSystemHelper.addSystem('clientId', 'system1', [regime])
+
+    invoice = await NewInvoiceHelper.add()
   })
```

Comment:
Whilst you are in there making changes I think you can make another slight tweak

```javascript
describe('Presroc Invoices controller', () => {
  let server
  let authToken
  let invoice

  beforeEach(async () => {
    await DatabaseHelper.clean()
    server = await init()

    const regime = await RegimeHelper.addRegime('wrls', 'WRLS')
    await AuthorisedSystemHelper.addSystem('clientId', 'system1', [regime])

    invoice = await NewInvoiceHelper.add()
  })
```
Basically, we don't use the `regime` anywhere so we don't need to store it in a `let`.